include/oc_ri.h

```diff
@@ -414,6 +432,15 @@ void oc_ri_remove_timed_event_callback(void *cb_data,
  */
 int oc_status_code(oc_status_t key);
 
+/**
+ * @brief convert the status code to string
+ *
+ * @param[in] key key the application level key of the code
+ * @return char* CoAP status code string
+ */
+/*OC_API*/
```

Comment:
I know how to fix this for SWIG, see: https://github.com/iotivity/iotivity-lite/pull/305/files#diff-0e60995c1ffb3932d92f32aa8b29750edc430eca97ee060bae91f74afa56d3df .

The issue is that the SWIG preprocessor doesn't include headers from other header files (in this case OC_API in oc_ri.h is defined in util/compiler.h, but the include will be ignored), so we need to either include oc_compiler.h or define OC_API in the .i file before oc_ri.h is included.